README.md

```diff
@@ -1,2 +1,133 @@
+
+#### export_accounts
+
+```bash
+> stellar-etl export_accounts --end-ledger 500000 --output exported_accounts.txt
+```
+
+This command exports accounts, starting from the genesis ledger and ending at the ledger determined by `end-ledger`. This command exports the point in time state of accounts, meaning that the exported data represents the account information as it was at `end-ledger`.
+
+#### export_offers
+
+```bash
+> stellar-etl export_offers --end-ledger 500000 --output exported_offers.txt
+```
+
+This command exports offers, starting from the genesis ledger and ending at the ledger determined by `end-ledger`. This command exports the point in time state of offer, meaning that the exported data represents the offerbook as it was at `end-ledger`.
+
+#### export_trustlines
+
+```bash
+> stellar-etl export_trustlines --end-ledger 500000 --output exported_trustlines.txt
+```
+
+This command exports trustlines, starting from the genesis ledger and ending at the ledger determined by `end-ledger`. This command exports the point in time state of trustlines, meaning that the exported data represents the trustline information as it was at `end-ledger`.
+
+### History Archive Commands
+
+These commands export information using the history archives. This allows users to provide a start and end ledger range. The commands in this category export a list of everything that occurred within the provided range.
+#### export_ledgers
+
+```bash
+> stellar-etl export_ledgers --start-ledger 1000 \
+--end-ledger 500000 --output exported_ledgers.txt
+```
+
+This command exports ledgers within the provided range. Both `start-ledger` and `end-ledger` are included in the export process.
+
+#### export_transactions
+
+```bash
+> stellar-etl export_transactions --start-ledger 1000 \
+--end-ledger 500000 --output exported_transactions.txt
+```
+
+This command exports transactions within the provided range. This range is inclusive.
+
+#### export_operations
+
+```bash
+> stellar-etl export_operations --start-ledger 1000 \
+--end-ledger 500000 --output exported_operations.txt
+```
+
+This command exports transactions within the provided range. This range is inclusive.
+
+### Stellar Core Commands
+
+These commands require a Stellar Core instance that is v13.2.0 or later. The commands use the Core instance to retrieve information about changes from the ledger. These changes can be in the form of accounts, offers, or trustlines.
+
+As the Stellar network grows, the Stellar Core instance has to catch up on an increasingly large amount of information. This can add some overhead to the commands in this category.
```

Comment:
"Some overhead" make sense but is underspecified. How does this overhead manifest? What should the end user do when seeing this overhead?